tests/plugins/test_pull_base_image.py

```diff
@@ -327,10 +327,24 @@ def test_manifest_list_verified(self, caplog):
                                     workflow_callback=self.prepare)
         assert log_message in caplog.text()
 
+    def test_manifest_list_verified_check_failed(self, caplog):
```

Comment:
The name of this test is misleading because the verification was actually successful, but it used the "fallback" mechanism of checking `orchestrate_build` plugin parameters. Maybe the name `test_fallback_to_orchestrate_build` is more descriptive?

Reply:
Fixed.